indico/modules/rb_new/schemas.py

```diff
@@ -70,7 +70,7 @@ class Meta:
         fields = ('id', 'name', 'capacity', 'building', 'floor', 'number', 'is_public', 'location_name', 'full_name',
                   'comments', 'division', 'is_reservable', 'reservations_need_confirmation', 'sprite_position',
                   'surface_area', 'latitude', 'longitude', 'telephone', 'key_location', 'max_advance_days',
-                  'owner_name', 'available_equipment', 'has_photo')
+                  'owner_name', 'available_equipment', 'has_photo', 'notification_for_assistance')
```

Comment:
this doesn't belong in the core

Reply:
So I think I should have a separate table in the plugin with a single column storing the ids of the rooms that allow to request assistance, right?

Reply:
I'd just use a plugin setting storing a list of room ids

Reply:
See `RoomConverter` in the cronjobs plugin

Reply:
Event better, thanks!